src/main/java/com/starrocks/connector/flink/cdc/DatabaseSync.java

```diff
@@ -79,6 +80,10 @@ public void create(StreamExecutionEnvironment env, String database, Configuratio
         this.excludingPattern = excludingTables == null ? null : Pattern.compile(excludingTables);
         this.sinkConfig = sinkConfig;
         this.tableConfig = tableConfig == null ? new HashMap<>() : tableConfig;
+
+        if (!this.tableConfig.containsKey(FAST_SCHEMA_EVOLUTION)) {
```

Comment:
If StarRocks version is before 3.2, this will lead to create table fail

Reply:
thoughtful，replace it with boolean. this boolean will control whether to execute schemachange